docs/sphinx/examples/python/tutorials/vqe.ipynb

```diff
@@ -53,25 +89,96 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def to_minimize(theta): \n",
+    "    \n",
+    "    exp_val = cudaq.observe(kernel, hamiltonian, theta).expectation()\n",
     "\n",
+    "    return exp_val \n",
+    "\n",
+    "\n",
+    "exp_vals = []\n",
+    "def callback(xk): \n",
+    "    exp_vals.append(to_minimize(xk)) \n",
+    "\n",
+    "\n",
+    "x0 = np.random.uniform(low = 0, high = 2*np.pi, size = parameter_count)\n",
+    "\n",
+    "result = minimize(to_minimize, x0, method='COBYLA', callback = callback, options= {'maxiter': 40})\n",
+    "\n",
+    "plt.plot(exp_vals)\n",
+    "plt.xlabel('Epochs')\n",
+    "plt.ylabel('Energy')\n",
+    "plt.title('VQE')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use the builtin optimizers within CUDAQ for the minimization procedure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.1371756094989427\n"
+     ]
+    }
+   ],
+   "source": [
     "optimizer = cudaq.optimizers.COBYLA()\n",
     "\n",
+    "\n",
     "energy, parameters = cudaq.vqe(kernel,\n",
-    "                               molecule,\n",
+    "                               hamiltonian,\n",
     "                               optimizer,\n",
     "                               parameter_count=parameter_count)\n",
     "\n",
     "print(energy)"
    ]
   },
   {
-   "attachments": {},
-   "cell_type": "markdown",
+   "cell_type": "code",
```

Comment:
I believe the concluding markdown comment about the energy convergence has been removed. If that was intentional, we should remove this empty code cell as well